src/test/ReactTestUtils.js

```diff
@@ -206,7 +209,15 @@ var ReactTestUtils = {
     );
     var internalInstance = ReactInstanceMap.get(inst);
     if (internalInstance && typeof internalInstance.tag === 'number') {
-      return findAllInRenderedFiberTreeInternal(internalInstance, test);
+      var fiber = internalInstance;
+      var root = fiber;
+      while (root.return) {
+        root = root.return;
+      }
+      var isRootCurrent = root.tag === HostContainer && root.stateNode.current === root;
```

Comment:
I'm not sure if this is a good way to tell that we're in a current tree. Any pitfalls?